src/dll/edbgEcmdDll.C

```diff
@@ -1541,11 +1541,17 @@ uint32_t dllGetMemProc(const ecmdChipTarget & i_target, uint64_t i_address, uint
 
   // Allocate a buffer to receive the data
   buf = (uint8_t *)malloc(i_bytes);
+  
+  // Set the block size
+  uint32_t blockSize = 0;
+  
+  //default cache inhibit is false
+  bool ci = false;
+
 
   // Make the right call depending on the mode
   if (i_mode == MEMPROC_CACHE_INHIBIT) {
-    // Set the block size
-    uint32_t blockSize = 0;
```

Comment:
How does ecmd deal with requests to cache-inhibited memory that may only support specific read sizes? I seem to recall that during bring-up it was possible to checkstop the hardware doing say a 4-byte cache-inhibited read to MMIO that only supported 8-byte reads (or vice-versa). Pdbg provides the user a way to explicitly specify this, but how does ecmd work? Does it just infer it somehow? We want to make sure we use the same inference logic here if that is the case.

Reply:
Well - ecmd doesn't handle any of that :)  It's just the API and it passes along the request to this level of code.  At the API level the user can specify the number of bytes to read (`i_bytes`), then we translate it here to blocksize.

I remember we had some issues getting Rick's tools going, but I no longer recall the details.  If there were special case things to handle or error check, we could do that here.

Reply:
If we need to change the behavior of the block size determination code, we can do that via another changeset/PR.  We'll leave this one focused on the adu->mem change and preserve the current behavior that has worked to date.